lib/src/actors_base.dart

```diff
@@ -11,8 +11,11 @@ class _Message {
   final int id;
   final content;
   final bool isError;
+  final String stackTraceToStringOrNull;
```

Comment:
> In fact, I would suggest naming it stackTraceAsString as that makes it clearer IMO what it is.

I do not agree and yes orNull will not be necessary as soon as nnbd lands but I'll make the change.

Also, it's called stackTraceToString because it is literally the toString version of the StackTrace which is the only representation that can be translated back into a StackTrace. asString tells the consumer it's just a string and nothing else.

Reply:
The problem with `stackTraceToString` it that it uses a known pattern: `xToY` is a pattern used by conversion functions from x to y.
`asString` tells that it is a type converted to a String, which is the exact meaning here.

Reply:
Of course, conventions take precedence over vague contextual information. Thank you for the clarification I failed to consider that.